bittide/src/Bittide/Node.hs

```diff
@@ -196,15 +202,34 @@ managementUnit ::
   -- ( Outgoing 'Bittide.Link'
   -- , Outgoing @Vector@ of master busses)
   ( Signal dom (DataLink 64)
-  , Vec nodeBusses (Signal dom (WishboneM2S (32 - CLog 2 (nodeBusses + 8)) 4 (Bytes 4)))
+  , Vec nodeBusses (Signal dom (WishboneM2S (NmuRemBusWidth nodeBusses) 4 (Bytes 4)))
   )
-managementUnit (ManagementConfig linkConfig peConfig) linkIn nodeS2Ms =
+managementUnit (ManagementConfig scatterConfig gatherConfig peConfig) linkIn nodeS2Ms =
   (linkOut, nodeM2Ss)
  where
-  (suS2M, nmuS2M0) = linkToPe linkConfig linkIn sc suM2S nmuM2S0
-  (linkOut, guS2M, nmuS2M1) = peToLink linkConfig sc guM2S nmuM2S1
-  (suM2S, (guM2S, rest)) = (head &&& (tail >>> (head &&& tail))) nmuM2Ss
-  (splitAtI -> (nmuM2S0, nmuM2S1), nodeM2Ss) = splitAtI rest
+  (suS2M, nmuS2M0) = scatterUnitWb scatterConfig nmuM2S0 linkIn suM2S
+  (linkOut, guS2M, nmuS2M1) = gatherUnitWb gatherConfig nmuM2S1 guM2S
+  (vecToTuple -> (suM2S, guM2S), rest) = splitAtI nmuM2Ss
+  (vecToTuple -> (nmuM2S0, nmuM2S1), nodeM2Ss) = splitAtI rest
   (_, nmuM2Ss) = toSignals (processingElement peConfig) (pure $ JtagIn low low low, nmuS2Ms)
-  nmuS2Ms = suS2M :> guS2M :> nmuS2M0 ++ nmuS2M1 ++ nodeS2Ms
-  sc = sequenceCounter
+  nmuS2Ms = suS2M :> guS2M :> nmuS2M0 :> nmuS2M1 :> nodeS2Ms
+
+-- managementUnitC ::
+--   forall dom nodeBusses nmuRemBusWidth .
+--   (KnownNat nmuRemBusWidth, 2 <= nmuRemBusWidth, HiddenClockResetEnable dom) =>
+--   -- |
+--   -- ( Configures all local parameters
+--   -- , Incoming 'Bittide.Link'
+--   -- , Incoming @Vector@ of master busses
+--   -- )
+--   GppeConfig nmuRemBusWidth ->
+--   Circuit
+--     (CSignal dom (DataLink 64))
+--     (CSignal dom (DataLink 64), Vec (32 - CLog 2 (nodeBusses + 8)) (Wishbone dom 'Standard 4 (Bytes 4)))
+-- managementUnitC (GppeConfig peConfig scatterConfig gatherConfig) = circuit $ \(linkIn, nmuWbs) -> do
+--   [wbScatCal, wbScat, wbGathCal, wbGu] -< nmuWbs
+--   jtag <- idleSource -< ()
+--   peWbs <- processingElement peConfig -< jtag
+--   linkOut <- gatherUnitWb gatherConfig -< (wbGathCal, wbGu)
+--   scatterUnitWb scatterConfig -< (linkIn, wbScatCal, wbScat)
+--   idC -< linkOut
```

Comment:
Is this not compiling yet? I'll give it a try locally, just commenting here so I don't forget